src/stim300_driver_node.cpp

```diff
@@ -22,15 +22,16 @@ Eigen::Quaterniond estimate_orientation(const Eigen::Vector3d& acc)
 
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, "stim300_driver_node");
+  std::string node_name = "stim300_driver_node";
+  ros::init(argc, argv, node_name);
 
   ros::NodeHandle node;
   std::string imu_path;
   std::string imu_link;
   std::string imu_output;
-  node.param<std::string>("device_path", imu_path, "/dev/ttyUSB0");
-  node.param<std::string>("imu_frame", imu_link, "imu_link");
-  node.param<std::string>("imu_output", imu_output, "/stim300_imu_fb");
+  node.param<std::string>((node_name + "/device_path"), imu_path, "/dev/ttyUSB0");
+  node.param<std::string>((node_name + "/imu_frame"), imu_link, "sam/imu_link");
+  node.param<std::string>((node_name + "/imu_output"), imu_output, "stim_imu");
```

Comment:
I belive this code will not work if the node is given another name by the launch file.

Reply:
E.g. if you call it just `name:=imu_driver` in the node tag